src/legacy/components/ImportTable.component.js

```diff
@@ -682,26 +682,20 @@ class ImportTable extends React.Component {
 
     render() {
         const { onRequestClose } = this.props;
-        const { infoDialog, users, isLoading, existingUsernames, allowOverwrite, areUsersValid, isImporting } =
-            this.state;
+        const { infoDialog, isLoading, isImporting } = this.state;
         const { multipleSelector, orgUnitRoots } = this.state;
-
-        const duplicatedUsernamesExist = this.getDuplicatedUsernamesExist(users, existingUsernames);
-        const showProcessButton = !users.isEmpty() && areUsersValid;
-        const actions = this.getActionsByState(allowOverwrite, duplicatedUsernamesExist, showProcessButton);
```

Comment:
Please clean-up unused methods like this. Also the ``Toggle`` that appeared next to the buttons should be added to the dialog content.

```
                {showOverwriteToggle && !templateUser && (
                    <Toggle
                        label={this.t("overwrite_existing_users")}
                        labelPosition="right"
                        toggled={allowOverwrite}
                        onToggle={this.toggleAllowOverwrite}
                        style={styles.overwriteToggle}
                    />
                )}
```